src/core/tests/widgets/test_webview.py

```diff
@@ -40,3 +41,18 @@ def test_set_content_invokes_impl_method(self):
 
         self.web_view.set_content(root_url, new_content)
         self.assertActionPerformedWith(self.web_view, 'set content', root_url=root_url, content=new_content)
+
+    def test_get_dom(self):
+        dom = self.web_view.dom
+        self.assertEqual(dom, 'DUMMY DOM')
```

Comment:
This should validate the `get_dom` call has been passed down to the implementation.